CHANGELOG.md

```diff
@@ -5,16 +5,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- [#245] getting stuck in status "starting" when a dogu cannot be started
```

Comment:
```suggestion
- [#245] Check Replica count instead of readiness status at Dogu startup to prevent blocking operator when a Dogu cannot be started
```
Maybe include a description of the solution you created, because the problem is already described in the issue (245).